src/commands/ssh.ts

```diff
@@ -95,20 +96,24 @@ export const sshCommand = (yargs: yargs.Argv) =>
  * - AWS EC2 via SSM with Okta SAML
  */
 const sshAction = async (args: yargs.ArgumentsCamelCase<SshCommandArgs>) => {
-  // Prefix is required because the backend uses it to determine that this is an AWS request
-  const authn = await authenticate();
+  try {
+    // Prefix is required because the backend uses it to determine that this is an AWS request
+    const authn = await authenticate();
 
-  const { request, privateKey, sshProvider } = await prepareRequest(
-    authn,
-    args,
-    args.destination
-  );
+    const { request, privateKey, sshProvider } = await prepareRequest(
+      authn,
+      args,
+      args.destination
+    );
 
-  await sshOrScp({
-    authn,
-    request,
-    cmdArgs: args,
-    privateKey,
-    sshProvider,
-  });
+    await sshOrScp({
+      authn,
+      request,
+      cmdArgs: args,
+      privateKey,
+      sshProvider,
+    });
+  } catch (error) {
+    throw errorBoundary(error, {}, args.debug);
```

Comment:
Currently added boundary only for SSH command. Once everything looks good, will extend to other commands.